mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1016,6 +1016,26 @@ func resourceContainerCluster() *schema.Resource {
 				ConflictsWith: []string{"enable_autopilot"},
 			},
 
+<% unless version == "ga" -%>
+			"node_pool_auto_config": {
+				Type:             schema.TypeList,
+				Optional:         true,
+				DiffSuppressFunc: containerClusterNodePoolAutoConfigDiffSuppress,
+				MaxItems:         1,
+				Description:      `Node pool configs that apply to all auto-provisioned node pools in autopilot clusters and node auto-provisioning enabled clusters.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"tags": {
```

Comment:
If I'm reading this right, this it putting "tags" directly inside "node_pool_auto_config". It's a bit of a pain, but could you include the "network_tags" field? This is important in case the API decides to add a new key inside network_tags as a sibling to tags, and/or a new key inside node_pool_auto_config as a sibling to network_tags.

Reply:
Ah yeah, fair enough. Refactored that in this last push.